### PR TITLE
Remove mdc mss

### PR DESCRIPTION
**Pull request by wei-lee**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,8 +3,8 @@ NAMESPACE=integreatly
 PROJECT=integreatly-operator
 REG=quay.io
 SHELL=/bin/bash
-PREVIOUS_TAG=1.13.2
-TAG=1.14.0
+PREVIOUS_TAG=1.14.0
+TAG=1.14.1
```
Comment (philbrookes) [inline Makefile]: we don't do bumps on each PR anymore

Comment (wei-lee) [reply Makefile]: @philbrookes I have reverted the changes to bump the version.

**Comment by grdryn** [comment]
@philbrookes does that ci/prow/e2e job usually pass?

**Comment by philbrookes** [comment]
@grdryn yeah, we rely on it passing before we merge anything

**Comment by wei-lee** [comment]
@philbrookes If you take a look at the errors, it says "time out". Did you see the error before? Do you know how can I fix the errors?

**Comment by philbrookes** [comment]
the e2e test can be tough to diagnose. Can you try running make test/e2e on your own cluster, so we can get a better idea of what is failing?

**Comment by philbrookes** [comment]
this has worked for me locally. Can we get this rebased against #251 and retry the e2e tests

**Comment by wei-lee** [comment]
@philbrookes the e2e test passed!

**Pull request merged**